lib/charms/mongodb/v0/mongodb_secrets.py

```diff
@@ -0,0 +1,125 @@
+"""Secrets related helper classes/functions."""
```

Comment:
This is an excellent file. Just so you know we have the VM Charm for MongoDB own the shared libraries, so in our development we try to create/change libraries on the MongoDB VM charm and only ever retrieve libraries on the MongoDB K8s charm. What is more is that I am fairly certain we do not allow publishing of libraries on the CI of K8s charms. 

With that said, I am personally okay with you making this change here since you have already written this excellent code! But I ask that after this PR merges that:
1. You create a follow up PR that adds this library to the MongoDB VM charm (I will immediately approve if the PR has **only this library**)
2. In the future if you would like to make changes to this library that you make them on the VM charm

Reply:
That sounds ALL wonderful.... 
I would like to do these changes equally on the VM as well anyway ;-)